vars/skipStage.groovy

```diff
@@ -102,7 +102,7 @@ boolean skip_ftest_hw(String size, String target_branch, String tags) {
         return false
     }
     String distro = (hwDistroTarget(size) =~ /([a-z]+)(\d+)(\.\d+)?/)[0][1..2].join('')
-    return !paramsValue('CI_' + size.replace('-', '_') + '_TEST', true) ||
+    return !paramsValue('CI_' + size + '_TEST', true) ||
```

Comment:
Has this always been true though?  I suspect to ensure backwards compatibility you want to test both:

```suggestion
    return !paramsValue('CI_' + size + '_TEST', true) ||
           !paramsValue('CI_' + size.replace('-', '_') + '_TEST', true) ||
```